### DISCUSSION TITLE
Resize animation canvas

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#179] [Ideas]
As far as I know it is not possible to resize the canvas of an animation but sometimes I want to like in my walk cycle which I posted before the hair goes off the page and I don’t want this. Could you add a way to resize the canvas of an animation.

**Comment by Kosdaos1** [comment]
Probably this question has already been raised many times, but this is really something that is very lacking - there is no way to change the size of the canvas already while working on animation. 

Plus, it would be possible to add a function of fields, as it is done in Roughanimator. Although I know that this problem is solved by zooming the camera - it's not always convenient:( 

And another small detail - when playing, you can darken what is not included in the camera. 

Thanks !

**Comment by DWrightAnimation** [comment]
On the bright side, at least it supports canvas sizes for 4K resolution, which is used in the show "Harley Quinn" I believe.
